svc-composition-service/app/db/persistant.py

```diff
@@ -0,0 +1,145 @@
+from redis import StrictRedis
+from config.config import PLUGIN_CONFIG
+
+import logging
+
+
+class RedisClient():
+    def __init__(self, **kwargs):
+
+        self.host = None
+        self.port = None
+        if ":" in PLUGIN_CONFIG["RedisAddress"]:
+            self.host, self.port = PLUGIN_CONFIG["RedisAddress"].split(":")
+        self.connection = None  # database
+        self._db_connection()
+
+    def __del__(self):
+        self.connection = None
+
+    def _db_connection(self):
+        try:
+            if self.host is not None and self.port is not None:
+                self.connection = StrictRedis(
+                    host=self.host,
+                    port=self.port,
+                    db=PLUGIN_CONFIG["Db"],
+                    socket_timeout=PLUGIN_CONFIG["SocketTimeout"])
+            else:
+                # default host = "localhost", port = 6379
+                self.connection = StrictRedis(
+                    db=PLUGIN_CONFIG["Db"],
+                    socket_timeout=PLUGIN_CONFIG["SocketTimeout"])
+        except Exception as err:
+            logging.error(
```

Comment:
Please make sure the logs are redirected to the log file defined in the start script.